cli/translator.js

```diff
@@ -0,0 +1,45 @@
+import {stat} from 'fs/promises'
+import {relative, resolve} from 'path'
+
+export default async function translator(path, root = ``) {
+  const appRoot = root || await searchRootFolder(resolve())
```

Comment:
`resolve()` is just the same thing as `cwd()`. We don't need them! :)

I want to keep our modules as simple as possible.

One thing that makes functions simple is that they only use the provided arguments and return value. I.e., no side effects/side loading.

Why not just do `searchRootFolder(path)`?

Reply:
DONE